packages/components/src/toggle-group-control/types.ts

```diff
@@ -98,7 +104,9 @@ export type ToggleGroupControlProps = Omit<
 	/**
 	 * React children
 	 */
-	children: ReactNode;
+	children:
```

Comment:
This change is not necessary, but I thought it could be interesting to enforce only the expected children type via TypeScript